### ISSUE TITLE
Interceptor name must be keyword or nil

### ISSUE THREAD

**Issue by tcoakley1** [frankiesardo/pedestal-swagger#11]
Hey, just checked out v0.4.1, you are passing the `~name` var to `interceptor/handler`
```
(defmacro defhandler
  "A drop-in replacement for pedestal's equivalent interceptor. Makes
  it simple to attach a meta tag holding the interceptor swagger
  documentation."
  [name doc args & body]
  `(def ~name
     (annotate ~doc (interceptor/handler ~name (fn ~args ~@body)))))
```

Which is unable to compile because the interceptor name must be nil or a keyword

```
(defn interceptor-name
  [n]
  (if-not (or (nil? n) (keyword? n))
    (throw (ex-info "Name must be keyword or nil" {:name n}))
    n))
```

Same goes for other macros in the `pedestal.swagger.core` namespace. 
Also it would be good if you updated the sample-project to use v0.4.1.

**Comment by frankiesardo** [comment]
Thanks. v0.4.1 was not really ready to be released. There will be a v0.4.2 soon that fixes that problem and introduces other niceties

**Comment by tcoakley1** [comment]
Cool, thats good to hear, I'll look forward to seeing what is coming!

**Comment by frankiesardo** [comment]
Fixed in 0.4.2
